conf/messages.en

```diff
@@ -56,6 +56,19 @@ nonOrgGgwPage.ifYouHaveOrgAccount.1=If you have another Government Gateway user
 nonOrgGgwPage.ifYouHaveOrgAccount.2=return to the sign in page
 nonOrgGgwPage.ifYouHaveOrgAccount.3=and enter the Government Gateway ID for your organisation.
 
+## Individual GGW Page
```

Comment:
Need Welsh entries in `messages.cy` else displays message keys in page when Welsh version shown ... 

Maybe simply rename this English file to `messages` so that it will act as a default...?